files/ie10-viewport/info.ini

```diff
@@ -0,0 +1,5 @@
+author = "Twitter"
```

Comment:
The original file is maintained by bootstrap, here:

https://github.com/twbs/bootstrap/blob/master/docs/assets/js/ie10-viewport-bug-workaround.js

The [thekondrashov](https://github.com/thekondrashov/ie10-viewport) repo has all of the CSS/JS for the IE10 workaround in one place.